vendor/bat-native-confirmations/src/bat/confirmations/internal/country_codes.h

```diff
@@ -0,0 +1,208 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_CONFIRMATIONS_INTERNAL_COUNTRY_CODES_H_
+#define BAT_CONFIRMATIONS_INTERNAL_COUNTRY_CODES_H_
+
+#include <set>
+#include <string>
+
+namespace confirmations {
+
+const std::set<std::string> kLargeAnonymityCountryCodes = {
```

Comment:
Read above comment.
constexpr std::array<const char*, N> kLargeAnonymityCountryCodes?

Reply:
I raised https://github.com/brave/brave-browser/issues/9164 to fix this globally in Brave Ads